CPAC/registration/registration.py

```diff
@@ -2450,13 +2450,13 @@ def register_ANTs_EPI_to_template(wf, cfg, strat_pool, pipe_num, opt=None):
         'functional_registration']['EPI_registration']['ANTs'][
         'interpolation']
 
-    node, out = strat_pool.get_data('sbref')
```

Comment:
Yes this was causing an issue with the rodent pipeline.

Reply:
This node block's docstring says https://github.com/FCP-INDI/C-PAC/blob/0265baf916a3b84467055f0c136b6875aeef0d37/CPAC/registration/registration.py#L2413-L2414 but `desc-preproc_bold` would be a 4-D file, right? And `sbref` is 3-D?

[Looks like](https://github.com/FCP-INDI/C-PAC/commit/54681d14f8af84d18bc50f7190fe5951e7a2a724#diff-299f7bbeaba89cb93fca4663599221b5bddd42b0b8e40f6638496419bc80ab3aL2362-R2390) it was `["desc-reginput_bold", "desc-mean_bold"]` (3-D, right?) before it was `sbref` there, 
![https://github.com/FCP-INDI/C-PAC/commit/54681d14f8af84d18bc50f7190fe5951e7a2a724?diff=unified#diff-299f7bbeaba89cb93fca4663599221b5bddd42b0b8e40f6638496419bc80ab3aL2359-R2393](https://user-images.githubusercontent.com/5974438/201118782-270bc9cf-b562-4b3c-af40-7b4502c95d38.png)

So I thing probably a nodeblock upstream of this nodeblock in the rodent pipeline is missing its `sbref` output and the change should be there instead of here